semantic_segmentation/scripts/train.py

```diff
@@ -1,23 +1,78 @@
-import os
+# import os
 
-os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
-os.environ["CUDA_VISIBLE_DEVICES"] = "1"
+# os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+# os.environ["CUDA_VISIBLE_DEVICES"] = "1"
 
 import argparse
-from datetime import datetime
 
 import matplotlib.pyplot as plt
 import numpy as np
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks.early_stopping import EarlyStopping
+from pytorch_lightning.loggers.tensorboard import TensorBoardLogger
 import torch
 import torch.nn as nn
 import torch.optim as optim
-from augment import get_transform
-from dataset import Dataset
-from torch.utils.tensorboard import SummaryWriter
 from torchmetrics import Accuracy
-from tqdm import tqdm
 from unet_bn import U2NET_lite
 
+from augment import get_transform
+from dataset import Dataset
+
+
+class UNET_LT(pl.LightningModule):
```

Comment:
Let's change the name to something not related to the architecture
Something challenge related?

In the future we will change the net

Reply:
sure